### ISSUE TITLE
Header block adding extrainious BR

### ISSUE THREAD

**Issue by SgtOddball** [madebymany/sir-trevor-js#341]
I've noticed I'm getting a very strange behaviour from the header block.

Namely, when a saved header block is returned and rendered, it's then adding another BR tag outside the header that's not present in the original string. 

It's not happening to any other block from what I can see, though by the look of things it might be scribe related.

**Comment by ninjabiscuit** [comment]
Thanks for reporting. We'll take a look

**Comment by SgtOddball** [comment]
I've been having a further nosy around since it's driving me mad (along with the wandering format bar but that's another story).

And I've found from the scribe project they've referenced having a fix in place for firefox (which I admit i'm using v.38) for adding and removing the BR tags. 

It might be a case of it's a fix thats not needed anymore or that the removal part of the process isn't triggering correctly. I'll have a further play with it and let you know what I find out.

**Comment by SgtOddball** [comment]
It looks to be a problem with scribe as I've found what's causing it from the scribe version. It's got two code blocks that should be getting called at different steps but it's missing off one step but still applying the next step. 

This block isn't being triggered 
```javascript 
Scribe.prototype.getContent = function () {
    return this._htmlFormatterFactory.formatForExport(this.getHTML().replace(/<br>$/, ''));
};
```

....but this still is adding an extra BR tag
```javascript
Scribe.prototype.setContent = function (content) {
    if (!this.allowsBlockElements()) {
        content = content + '<br>';
    }
    this.setHTML(content);
    this.trigger('content-changed');
};
```
On a related note, from what I can see of the scribe project they have had some updates lately but it looks like this would still be an issue.

**Comment by lukaszsagol** [comment]
@SgtOddball could you check if using `scribe.getContent()` instead of `getHTML()` also causes the same problem?

**Comment by SgtOddball** [comment]
It's still doing it, or at least... it was until I tried something else. 

On the heading block ```scribeOptions: { allowBlockElements: false },``` on a whim as an experiment I changed it to 'true' (considering the elements it doesn't do it to where also set to true it seemed just crazy enough to work)  lo and behold... no more random BR elements. 

Not sure if you want to change the block setting or not but so far so good. Will test on other browsers too to see if it's had any other random effects.
